src/reader/components/ReaderLine.vue

```diff
@@ -5,7 +5,12 @@
     </section>
     <div class="line u-flex" v-else>
       <div class="line-ref">{{ line.ref }}</div>
-      <div class="line-text">
+      <div
+        class="line-text metrical"
+        v-if="metrical"
+        v-html="line.metricalAnnotations[0].htmlContent"
```

Comment:
This causes console errors if the given passage doesn't have metrical content, e.g.

https://deploy-preview-22--explorehomer-dev.netlify.app/reader?urn=urn%3Acts%3AgreekLit%3Atlg0012.tlg001.msA%3A1.1-1.25

![image](https://user-images.githubusercontent.com/629062/84203840-b865ea00-aa6f-11ea-8bd8-906268992c53.png)

![image](https://user-images.githubusercontent.com/629062/84203853-bc920780-aa6f-11ea-9993-0e8411a2a87a.png)

maybe add something to check the length?

Reply:
@jacobwegner fixed